README.md

```diff
@@ -177,44 +177,39 @@ Example of a partial CR:
 
 ### Mounting and injecting binding information
 
-This specification allows for data to be mounted using volumes or injected using environment variables.  The best practice is to mount any sensitive information, such as passwords, since that will avoid accidentally exposure via environment dumps and subprocesses.  Also, binding binary data (e.g. .p12 certificate for Kafka) as an environment variable might cause a pod to fail to start (stuck on `CrashLoopBackOff`), so it advisable for backing services with such binding data to mark it with `bindAs: volume`.
+#### Mounting data
+Implementations of this specification **MUST** mount the binding data into the consuming application container in the following location:
+
+```
+<mountPathPrefix>/<service-id>/secret/<persisted_secret>
+```
+
+Where:
+* `<mountPathPrefix>` defaults to `/platform/bindings` if not specified in the `ServiceBinding` CR via the `mountPathPrefix` element.  The environment variable `SERVICE_BINDINGS_ROOT` **MUST** be set to the value of `<mountPathPrefix>`, so that applications can always find this directory.
```

Comment:
- What are your thoughts on making `SERVICE_BINDINGS_ROOT` available in `<mountPathPrefix>/<service-id>/metadata/envVars` in addition to env var?
- Should `SERVICE_BINDINGS_ROOT` be available as a key-value pair in the binding secret?
- Do you think the spec should mention anything about when there are multiple `BindingRequest` CRs with conflicting `mountPathPrefix`?

Reply:
I wonder if that would be redundant, since to reach `<mountPathPrefix>/<service-id>/metadata/envVars` the app would already know the value of `SERVICE_BINDINGS_ROOT`?   I think the pattern is:  the app code fetches `SERVICE_BINDINGS_ROOT` and then crawls through the bindings there.

Reply:
no harm in including it though

Reply:
> ...since to reach `<mountPathPrefix>/<service-id>/metadata/envVars` the app would already know the value of `SERVICE_BINDINGS_ROOT`...

That's an important point that I missed lol... I agree that it's redundant and no point in adding